scheduler/src/cook/mesos/util.clj

```diff
@@ -24,44 +24,58 @@
             [metatransaction.core :refer (db)]
             [metrics.timers :as timers]
             [plumbing.core :as pc :refer (map-vals map-keys)])
-  (:import [java.util Date]))
+  (:import [java.util Date]
+           [com.google.common.cache CacheBuilder]
+           [java.util.concurrent TimeUnit]))
```

Comment:
please lexicographically sort the `:import`s

Reply:
ACK